pkg/asset/machines/gcp/machines.go

```diff
@@ -105,7 +105,11 @@ func provider(clusterID string, platform *gcp.Platform, mpool *gcp.MachinePool,
 }
 
 // ConfigMasters assigns a set of load balancers to the given machines
-func ConfigMasters(machines []machineapi.Machine, clusterID string) {
+func ConfigMasters(machines []machineapi.Machine, clusterID string, publish types.PublishingStrategy) {
+	if publish == types.InternalPublishingStrategy {
+		return
```

Comment:
the return here is a little weird, because we plan to add the instance groups soon in case of internal..
so something like https://github.com/openshift/installer/blob/a932f024e7db4dae6c4ee0b454fece780204c0e5/pkg/asset/machines/aws/machines.go#L148-L163 is more appropriate ?

Reply:
changed, let me know if you like it better now